src/meta-srv/src/service/heartbeat.rs

```diff
@@ -111,13 +111,10 @@ impl heartbeat_server::Heartbeat for Metasrv {
                 }
             }
 
-            info!(
-                "Heartbeat stream closed: {:?}",
-                pusher_key.as_ref().unwrap_or(&"unknown".to_string())
-            );
+            info!("Heartbeat stream closed: {:?}", pusher_id);
```

Comment:
Can we use display?
```suggestion
            info!("Heartbeat stream closed: {}", pusher_id);
```

Reply:
`pusher_id` is an Option

Reply:
I've made `Debug` and `Display` the same format.